test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetTrustTests.cs

```diff
@@ -0,0 +1,508 @@
+
+                //Act
+                var result = _msbuildFixture.RunDotnet(
+                    pathContext.SolutionRoot,
+                    $"nuget trust list --configfile ..{Path.DirectorySeparatorChar}nuget.config",
+                    ignoreExitCode: true);
+
+                // Assert
+                result.Success.Should().BeTrue();
+                SettingsTestUtils.RemoveWhitespace(result.Output).Should().Contain(SettingsTestUtils.RemoveWhitespace(expectedAuthorContent));
+            }
+        }
+
+        [CIOnlyTheory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public async Task Trust_Author_RelativePathConfileFile_Success(bool allowUntrustedRoot)
+        {
+            // Arrange
+            var nugetConfigFileName = "NuGet.Config";
+            var package = new SimpleTestPackageContext();
+
+            using (SimpleTestPathContext pathContext = _msbuildFixture.CreateSimpleTestPathContext())
+            using (MemoryStream zipStream = await package.CreateAsStreamAsync())
+            using (TrustedTestCert<TestCertificate> trustedTestCert = SigningTestUtility.GenerateTrustedTestCertificate())
+            {
+                string certFingerprint = SignatureTestUtility.GetFingerprint(trustedTestCert.Source.Cert, HashAlgorithmName.SHA256);
+                string signedPackagePath = await SignedArchiveTestUtility.AuthorSignPackageAsync(trustedTestCert.Source.Cert, package, pathContext.PackageSource);
+                var config = $@"<?xml version=""1.0"" encoding=""utf-8""?>
+                    <configuration>
+                      <packageSources>
+                        <!--To inherit the global NuGet package sources remove the <clear/> line below -->
+                        <clear />
+                        <add key=""NuGetSource"" value=""{pathContext.PackageSource}"" />
+                       </packageSources>
+                      <config>
+                        <add key=""signaturevalidationmode"" value=""accept"" />
+                      </config>
+                      <trustedSigners>
+                      </trustedSigners>
+                    </configuration>";
+                SettingsTestUtils.CreateConfigurationFile(nugetConfigFileName, pathContext.WorkingDirectory, config);
+                var nugetConfigPath = Path.Combine(pathContext.WorkingDirectory, nugetConfigFileName);
+                var allowUntrustedRootArg = allowUntrustedRoot ? "--allow-untrusted-root" : string.Empty;
+                var allowUntruestedRootValue = allowUntrustedRoot ? "true" : "false";
+
+                // Act
+                var resultAdd = _msbuildFixture.RunDotnet(
+                    pathContext.SolutionRoot,
+                    $"nuget trust author nuget {signedPackagePath}  {allowUntrustedRootArg} --configfile {nugetConfigPath}",
+                    ignoreExitCode: true);
```

Comment:
why do these tests ignore the exit code? None of these commands should exit nonzero. `ignoreExitCode` only makes sense if you're testing failure cases.

Reply:
Very good point. I'm correcting for all my tests. 
Root cause was `I copied existing test` and reused it for all cases.